### PR TITLE
13-4-22 Content

### PR DESCRIPTION
**Pull request by umaaug**

### CODE REVIEW

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md [hunk 1]
```diff
@@ -36,45 +37,30 @@ Medieval digests like the Śuddhiprakāśa (pp 151–152) state that when a pers
...
+The sarvaprāyaścitta was to be performed by the dying man or by his son or other relative who was to shave himself, take a bath, drink _pañcagavya_, honour a brāhmana with sandal paste and other things, after having honoured the cow or the money to be paid in lieu of it, and after referring to the several kinds of sins that one may be guilty of (vide note[^429]) he is to make a gift of a cow (with a calf) to the brāhmana or in lieu of it, of some money. After sarvaprāyaścitta come the ten _dānas_ stated above. In the danas the Kāmastuti[^430] as studied by the followers of each veda was to be recited. The Garuḍapurāṇa (II. 4.7-9) further prescribes other dānas called mahā-dānas viẓ, of sesame, iron, gold, cotton, salt, seven kinds of corn, land and cows when a man is dying and also other gifts called padadānas such as of umbrella, sandals, ring, water-jar, seat, food.
```
Comment (vvasuki) [inline kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:48]: brāhmana should have ण

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/_index.md [hunk 2]
```diff
@@ -3,6 +3,6 @@ title = "+2 Antyeṣṭi, Āśauca, Śuddhi"
 
 +++
 
-SECTION II 
+SECTION II
 
-ANTYESTI (rites after death ), SUDDHI ( purification from impurity due to death, birth and other causos) 
+ANTYEṢTI (rites after death ), ŚUDDHI ( purification from impurity due to death, birth and other causes)
```
Comment (vvasuki) [inline kANe/v4/2_antyeShTi_Ashaucha_shuddhi/_index.md:8]: ANTYEṢTI should have टि

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md (hunk 1 above)
Comment (umaaug) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:48]: corrected.

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md [hunk 3]
```diff
@@ -36,45 +37,30 @@ Medieval digests like the Śuddhiprakāśa (pp 151–152) state that when a pers
...
+The Hir. P. S. I. 1 provides that when it is thought that the _āhitāgnı_ is going to die, he (the son or the like) should mutter in the dying man's ear (if he is a knower of Brahma) the two _anuvākas_ of the Tai. Up. II. 1 (beginning with 'brahmavidāpnoti param') and III. 1 (beginning with 'Bhrgurvai vāruniḥ'). The A. K. D. (P 18) says [^432] that if the dying man is unable to engage in _japa_ he should revolve in his mind the benign form of Visṇu or Śiva and should listen to the thousand names (of Visṇu or Śiva), the Bhagavadgitā, the Bhāgavata, Bhārata, Rāmāyana, the Upanisads like Īśāvāsya and hymns to Soma.
```
[the remark this answers is not in the document]
Comment (umaaug) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:63]: corrected

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/_index.md (hunk 2 above)
Comment (umaaug) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/_index.md:8]: means I should write ANTYEṢṬI like this?

Comment (vvasuki) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/_index.md:8]: Yes

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (vvasuki) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:63]: Please correct similarly for Rāmāyana as well.

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/_index.md (hunk 2 above)
Comment (umaaug) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/_index.md:8]: corrected

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md [hunk 4]
```diff
@@ -58,137 +58,108 @@ The sarvaprāyaścitta was to be performed by the dying man or by his son or oth
...
 
 448 This appears to postulate the acquisition of a new etbereal body for the departed enabling him to enjoy the pleasures of pitrloka.
 
 449. 'Savıdatran'who will kaow or recogoize you. The Nir, V1.14 explains 'sayıdatrah Lalyānavidyāh'.
 
-450. The life implored for here is that of the persons related to the deceased who are left behind on the earth. 'Asutrpa' according to Sayana And Oldenberg means 'who steal away the lives of men'. This is a good mcading 10 view of what is stated so the last pada of the verse. In trags 1asing as done above the word is taken as 'a+su+trpa', while Sāyana takes it as 'asu +- trpa'.
+450. The life implored for here is that of the persons related to the deceased who are left behind on the earth. 'Asutrpa' according to Sayana And OldenbeṚg means 'who steal away the lives of men'. This is a good mcading 10 view of what is stated so the last pada of the verse. In trags 1asing as done above the word is taken as 'a+su+trpa', while Sāyana takes it as 'asu +- trpa'.
```
Comment (vvasuki) [inline kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:158]: Should be Oldenberg

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (umaaug) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md]: corrected

File: kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md (hunk 4 above)
Comment (umaaug) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:158]: yes. i will correct that when i come to it.

Comment (umaaug) [reply kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md:158]: that was a wrong replacement. i will correct it.

**Pull request merged**